### DISCUSSION TITLE
Lots of Firestick Issues

### DISCUSSION THREAD

**Discussion by npentell** [jellyfin/jellyfin-androidtv#2908] [General]
I'm having a hard time with the firestick app here.  I recently switched from kodi to all the *arrs and added jellyfin as the frontend.  I love the web version and everything that has to do with jellyfin is wonderful....everything except the Firestick app.   I don't mean this to come off as rude but, as a project manager, I don't see how this was even released.  Some videos freeze while playing but the audio plays.  I have to go in to the setting and change the player back and fourth to get things to work.  Some videos play ok but the audio is off one way or another. I just had to go in and change the audio delay to -3010.  That's a massive amount. And some videos the sound doesn't work. Then I have to go in and downmix the audio to stereo when I intended for it to play on my surround sound. system.  I wouldn't give this a second thought any other time since I don't mind just using the web player but My wife and kids are firestick all the way and this needs to work for them.  The only reason I posted this is because these seem like dumpster fire issues and I see similar discussions from 2021 so are these audio/video issues being worked on? IS it not possible to have my computer do the transcoding before it sends it up?  I would appreciate any advice for this app.  Thanks a lot.

P.S. While I was writing this my audio for my show became out of sync half way though the episode and I had to stop and change it back from -3010 to 0

**Comment by npentell** [comment]
Edit. in the case of the audio out of sync, I tried the show on the web player and had the same sync issue.  I changed the audio delay to -3010 to get things in sync, a few min later I had to change it back to 0, then to +5000, then to +3000.  It's all over the place.  I see issues spanning back for years with .eac3 files and this happens to be one of them.

**Comment by nielsvanvelzen** [comment]
If you're not satisfied with my work (that I do for free in my spare time btw) you can use different software, or feel free to work on those issues yourself. I have no obligation nor motivation to help you with this attitude.

**Comment by npentell** [comment]
To begin with, I was under the impression that this was a company that
created this app and not a one-man show doing all the work.  I applaud you
for putting so much effort into this and being able to make it such a
popular app with such limited help.  Having said that, I wasn't intending
to be rude.  Other than the mention that I was surprised it had been
released from a project manager's viewpoint I was just using the discussion
as intended to make aware the issues that were happening in the Firestick
App. Thanks for all the hard work.

**Comment by jordanmichaelrushing** [comment]
Fyi this desync problem is still a problem. I code but idk anything about audio syncing, @nielsvanvelzen is there any chance this issue could be prioritized?
